script.js

```diff
@@ -1,51 +1,67 @@
 let cardObjectsArray = [
     {
-        frontOfCard : 'images/pinkHamster.jpg'
+        frontOfCard: 'images/pinkHamster.jpg',
+        pairID: 'pairOne'
```

Comment:
instead of a string like `'pairOne'` use a number like `1`

Reply:
wouldn't make any difference but usually ids are numbers, also change `pairID` to `pairId`, more conventional way of applying camel case

Reply:
this is done